### PR TITLE
Update statsd-mappings.yml

### PR DESCRIPTION
**Pull request by stuart23**
For tasks that belong to task groups, the emitted metric format is `airflow.dag.<dag_id>.<taskgroup_id>.<task_id>.duration` and therefore it is not being picked up by the existing airflow_task_duration mapping. This adds an extra mapping to capture those use metrics and parse them.

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by Taragolis** [comment]
https://github.com/apache/airflow/blob/35c1a1a024e652c001930bf42416e06d3c20cf95/chart/files/statsd-mappings.yml#L18-L23

**Comment by stuart23** [comment]
> https://github.com/apache/airflow/blob/35c1a1a024e652c001930bf42416e06d3c20cf95/chart/files/statsd-mappings.yml#L18-L23

This is fully backwards compatible, it will ship more metrics than already exist and won't change any existing metrics

File: chart/files/statsd-mappings.yml [hunk 1]
```diff
@@ -52,6 +52,13 @@ mappings:
       dag_id: "$1"
       task_id: "$2"
 
+  - match: airflow.dag.*.*.*.duration
```
Comment (jedcunningham) [inline chart/files/statsd-mappings.yml:55]: You can use dots in normal task_ids too. This doesn't mean the task is in a task group.

**Comment by jedcunningham** [comment]
> This is fully backwards compatible, it will ship more metrics than already exist and won't change any existing metrics

The chart just remaps, it doesn't exclude metrics that aren't covered. Task durations for tasks in task groups already are exposed. This would move them.

e.g.

```
airflow_dag_task_group_example_my_taskgroup_sometask_duration{quantile="0.5"} 0.17767495400000002
airflow_dag_task_group_example_my_taskgroup_sometask_duration{quantile="0.9"} 0.17767495400000002
airflow_dag_task_group_example_my_taskgroup_sometask_duration{quantile="0.99"} 0.17767495400000002
airflow_dag_task_group_example_my_taskgroup_sometask_duration_sum 0.17767495400000002
airflow_dag_task_group_example_my_taskgroup_sometask_duration_count 1
```

Would become:

```
airflow_task_duration{dag_id="task_group_example",task_id="sometask",taskgroup_id="my_taskgroup",quantile="0.5"} 0.085300168
airflow_task_duration{dag_id="task_group_example",task_id="sometask",taskgroup_id="my_taskgroup",quantile="0.9"} 0.085300168
airflow_task_duration{dag_id="task_group_example",task_id="sometask",taskgroup_id="my_taskgroup",quantile="0.99"} 0.085300168
airflow_task_duration_sum{dag_id="task_group_example",task_id="sometask",taskgroup_id="my_taskgroup"} 0.085300168
airflow_task_duration_count{dag_id="task_group_example",task_id="sometask",taskgroup_id="my_taskgroup"} 1
```

File: chart/files/statsd-mappings.yml (hunk 1 above)
Comment (stuart23) [reply chart/files/statsd-mappings.yml:55]: So the 2 dots will not be able to glob those tasks either then?

Comment (jedcunningham) [reply chart/files/statsd-mappings.yml:55]: Another curve ball, guess what? Dags can have dots too :) So imagine we have `airflow.dag.first.second.third`. This could be:

- dag_id=first, task_group=second, task_id=third
- dag_id=first.second, task_id=third (no task_group at all)
- dag_id=first, task_id=second.third (no task_group at all)

You can also disable the task group id prefixing with `prefix_group_id=False`, so those last 2 could actually be in a task group, but not one named `second`.

Basically, there isn't a reliable way to extract task group id out.
